ccp/apis/v1/ccp_server/backend/builds/build_logs.py

```diff
@@ -75,6 +73,10 @@ def response(namespace, appid, jobid, desired_tag, build):
                 p.desired_tag == desired_tag:
             prebuild_exists = p.pre_build_context and p.pre_build_script
             break
+
+    if not prebuild_exists:
```

Comment:
We initiated `prebuild_exists` with `prebuild_exists=False` and then the only operation it's undergoing is an `and` operation. What is the expected result of:
```python
prebuild_exists = p.pre_build_context and p.pre_build_script
```
and do we really need to do:
```python
if not prebuild_exists:
    prebuild_exists=False
```

Reply:
I also had this idea, here if the pre_build_context exists, the `and` operation returns `True` but in case of not existing it returns `None` and gets missing in the api return body. this extra check makes sure the variable is not having `None` value and present in API return body